samples/sample.yaml

```diff
@@ -56,23 +55,31 @@ reports:
       - name: "project-new"
         config:
           report-on: 'Epic'
-          daysAgo: 100      
+          daysAgo: 100
   - name: Issues
     title: "{{{ config.name }}} Project Report"
     kind: markdown 
     timezoneOffset: -8
     targets: []
     sections: 
       - name: "repo-issues"
-        targets: ['sanenodeRepo', 'quotesFeedRepo']          
+        targets: ['sanenodeRepo', 'quotesFeedRepo']
       - name: "repo-issues"
         targets: ['sanenodeRepo']
       - name: "repo-issues"
-        targets: ['quotesFeedRepo']        
+        targets: ['quotesFeedRepo']
   - name: QuotesFeedIssues
     title: "QuotesFeed Issues Report"
     kind: markdown 
     timezoneOffset: -8
     targets: ['quotesFeedRepo']
-    sections:           
-      - name: "echo-board" 
+    sections:
+      - name: "echo-board"
+  - name: Issues Cycle Time
+    title: "{{{ config.name }}} Project Report"
+    kind: markdown
+    timezoneOffset: -8
+    targets: ['todoProject']
+    sections:
+      - name: "issue-count-cycle-time"
```

Comment:
Add this section to the first report which already targets the todoProject project board.  That way the sample shows all of the project sections in one sample and all of the repo sections in one sample report.